src/main/java/com/sourcegraph/cody/agent/CodyAgentClient.java

```diff
@@ -18,7 +18,6 @@ public class CodyAgentClient {
 
   private static final Logger logger = Logger.getInstance(CodyAgentClient.class);
   @Nullable public CodyAgentServer server;
-  @Nullable public CodyAgentDocuments documents;
```

Comment:
this change is unrelated, right? is this field unused?

Reply:
Correct, unrelated, but I've created static factory method for TextDocument and it's turned out that most of usages are dead code. So i removed that (see: CodyAgentDocuments).